### DISCUSSION TITLE
ani-cli with rust cli tools?

### DISCUSSION THREAD

**Discussion by IndianaRoy** [pystardust/ani-cli#1146] [General] [answered]
Hey,

I love the tool, and found it really useful. Quick question, have anyone tried building it with rust tools?
The dependencies have some rewrites on rust that are generally faster, tools like:

- [ripgreg](https://github.com/BurntSushi/ripgrep/) instead of grep

- [sd](https://github.com/chmln/sd) instead of sed

- [skim](https://github.com/lotabout/skim) instead of fzf

**Comment by port19x** [comment, marked as answer]
Since ani-cli does network requests it's unlikely to benefit from the faster rust programs.
I'm pretty sure most of us maintainers use ripgrep, fd and exa tho

**Reply by IndianaRoy** [reply]
Yeah, It's just to reduce dependencies. I think this is my only dependency for fzf
